core/src/main/java/com/microsoft/applicationinsights/internal/config/ChannelXmlElement.java

```diff
@@ -36,7 +36,10 @@
 public class ChannelXmlElement {
```

Comment:
As far as I understand this class does not hold an XML element but the channel configuration, so I think it should be named accordingly